README.md

```diff
@@ -18,6 +20,23 @@ You can try out the user-side of Calypso on [WordPress.com](https://wordpress.co
 4.	Execute `make run` or `make dashboard` (for a more visually-oriented interface) from the root directory of the repository.
 5.	Open `calypso.localhost:3000` in your browser.
 
+### Docker
+
+1. Download and install [Docker](https://docs.docker.com/engine/installation/)
+2. Add `127.0.0.1 calypso.localhost` to your local hosts file.
+2. Run `docker-compose up`
```

Comment:
We should probably move this to [this section](https://github.com/Automattic/wp-calypso/blob/master/docs/install.md#alternate-install-vagrant) where we mention alternatives - such as Vagrant - for installing Calypso.